lib/ask_web/controllers/folder_controller.ex

```diff
@@ -41,6 +41,7 @@ defmodule AskWeb.FolderController do
         where: f.project_id == ^project.id
       )
       |> Repo.all()
+      |> Repo.preload(:surveys)
```

Comment:
We actually did the exact opposite 1.5 years ago to fix a loading issue (loading too much is fine when you have a dozen surveys, but it's starting to take far too long when you start to have hundreds of them).

We might want to report how many running surveys a folder has, with a quick SQL query.

Also: a Folder can contain PanelSurveys that themselves can contain running Surveys, so you'll probably want to left join them too in the SQL query.

Reply:
`PanelSurvey` ids are stored in the `Survey` then just querying the surveys table should make it no? There is no `state` in the panel surveys to match `running` with, this is on the surveys.